pkg/metrics/exporter/snapshot/metrics.go

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2021. Alibaba Cloud. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package snapshot
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	daemonIDLabel = "daemon_id"
+	timeLabel     = "time"
+	eventLabel    = "event"
+)
+
+var (
+	NydusdEvent = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "nydusd_lifetime_event_times",
+			Help: "nydusd lifetime event times.",
+		},
+		[]string{daemonIDLabel, timeLabel, eventLabel},
+	)
```

Comment:
Snapshot package is an implementation of Containerd remote snapshotter. It does not know about nydusd daemons backing the nydus meta layer snapshot. Can we move the code to the daemon package?

Reply:
Yes, done.